settings.py

```diff
@@ -74,7 +74,7 @@
 CERTS_ARE_CALLED_PLURAL = 'certificates'
 
 # Programmatic disclaimer text
-CERTS_SITE_DISCLAIMER_TEXT = ''
+CERTS_SITE_DISCLAIMER_TEXT = '<b>PLEASE NOTE:</b> SOME ONLINE COURSES MAY DRAW ON MATERIAL FROM COURSES TAUGHT ON-CAMPUS BUT THEY ARE NOT EQUIVALENT TO ON-CAMPUS COURSES. THIS STATEMENT DOES NOT AFFIRM THAT THIS PARTICIPANT WAS ENROLLED AS A STUDENT AT STANFORD UNIVERSITY IN ANY WAY. IT DOES NOT CONFER A STANFORD UNIVERSITY GRADE, COURSE CREDIT OR DEGREE, AND IT DOES NOT VERIFY THE IDENTITY OF THE PARTICIPANT.'
```

Comment:
This line is pretty long, weighing in at 414 characters.

Could we wrap it in parens and split the strings?

```python
CERTS_SITE_DISCLAIMER_TEXT = (
    '<b>PLEASE NOTE:</b> SOME ONLINE COURSES MAY DRAW ON MATERIAL FROM COURSES '
    'TAUGHT ON-CAMPUS BUT THEY ARE NOT EQUIVALENT TO ON-CAMPUS COURSES.  '
    'THIS STATEMENT DOES NOT AFFIRM THAT THIS PARTICIPANT WAS ENROLLED '
    'AS A STUDENT AT STANFORD UNIVERSITY IN ANY WAY. '
    'IT DOES NOT CONFER A STANFORD UNIVERSITY GRADE, COURSE CREDIT OR DEGREE, '
    'AND IT DOES NOT VERIFY THE IDENTITY OF THE PARTICIPANT.'
)
```